external/storm-kafka-client/src/test/java/org/apache/storm/kafka/spout/test/KafkaSpoutTopologyMainNamedTopics.java

```diff
@@ -94,10 +98,10 @@ protected StormTopology getTopologyKafkaSpout() {
 
     protected KafkaSpoutConfig<String,String> getKafkaSpoutConfig() {
         ByTopicRecordTranslator<String, String> trans = new ByTopicRecordTranslator<>(
-                (r) -> new Values(r.topic(), r.partition(), r.offset(), r.key(), r.value()),
```

Comment:
`(Values & Serializable)(r) -> new Values(r.topic(), r.partition(), r.offset(), r.key(), r.value()` might work, but still think Func should be Serializable.